plasmapy/plasma.py

```diff
@@ -0,0 +1,79 @@
+"""
+plasmapy.plasma
+===============
+
+Defines the core Plasma class used by PlasmaPy to represent plasma properties.
+"""
+
+import numpy as np
+import astropy.units as u
+
+mu0 = np.pi * 4.0e-7 * (u.newton / (u.amp**2))
+
+class Plasma():
+    """Core class for describing and calculating plasma parameters.
+
+
```

Comment:
I left the blank line in to remind me to fill it with something, but time constraints prevented this. I know, I'm a terrible person.